test/browser/features/fixtures/packages/rollup.config.mjs

```diff
@@ -24,8 +26,15 @@ export default {
     ...(isCdnBuild ? cdnOutputOptions : {}),
   },
   plugins: [
-    nodeResolve({ browser: true, jail: path.resolve(`${__dirname}/..`) }),
-    commonjs()
+    nodeResolve({ browser: true, jail: path.resolve(`${__dirname}/..`), extensions: ['.mjs', '.js', '.json', '.node', '.jsx'] }),
+    commonjs(),
+    babel({ babelHelpers: 'bundled' }),
+    replace({
+      preventAssignment: true,
+      values: {
+        'process.env.NODE_ENV': JSON.stringify('production'),
+      },
+    })
```

Comment:
required when using react

Reply:
Now I'm looking at the vue integration I think it makes sense to move this into its own config for the react router fixture, possibly extending this rollup.config.mjs.